app/Repositories/LocationRepository.php

```diff
@@ -50,30 +50,33 @@ public function getData($location_id, $filter)
 
     public function getAllShift($location, $filter)
     {
-        $full_time_seat = $this->getByShift($location, $filter, config('site.shift.all'))->pluck('total', 'start')->toArray();
-        $morning = $this->getByShift($location, $filter, config('site.shift.morning'))->pluck('total', 'start')->toArray();
-        $afternoon = $this->getByShift($location, $filter, config('site.shift.afternoon'))->pluck('total', 'start')->toArray();
-        $total = $this->getTotalSeatData($filter, __('Total Seat'), config('site.analystic.total-color'), $location->total_seat);
-
-        $morning_data = $this->analystic($full_time_seat, $morning, $filter, __('1. Morning:'), config('site.analystic.default-color'));
-        $afternoon_data = $this->analystic($full_time_seat, $afternoon, $filter, __('2. Afternoon:'), config('site.analystic.afternoon-color'));
-
-        return array_merge($morning_data, $afternoon_data, $total);
-    }
-
-    public function getTotalSeatData($filter, $title, $className, $total_seat)
-    {
-        $dates = $this->getDatesOfFilter($filter);
-        $data = [];
-        foreach ($dates as $date) {
-            $data[] = [
-                'start' => $date,
-                'title' => $title . ': ' . $total_seat,
-                'className' => $className
-            ];
-        }
-
-        return $data;
+        $totalSeat = $location->total_seat;
+        $fullTimeSeat = $this->getByShift($location, $filter, config('site.shift.all'))
+            ->pluck('total', 'start')
+            ->toArray();
+        $morning = $this->getByShift($location, $filter, config('site.shift.morning'))
+            ->pluck('total', 'start')
+            ->toArray();
+        $afternoon = $this->getByShift($location, $filter, config('site.shift.afternoon'))
+            ->pluck('total', 'start')
+            ->toArray();
+
+        $morningData = $this->analystic(
+            $fullTimeSeat, 
+            $morning, 
+            $filter, 
+            __('1. Morning:'), 
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Repositories/LocationRepository.php</br>Whitespace found at end of line